apps/studio/src/components/tableinfo/TableRelations.vue

```diff
@@ -219,7 +219,27 @@ export default Vue.extend({
             // @ts-expect-error Incorrectly typed
             valuesLookup: this.getTables
           },
-          cellEdited: (cell) => cell.getRow().getCell('toColumn')?.setValue(null)
+          cellEdited: async (cell) => { 
+            const data = cell.getRow().getData()
+            const schema = data['toSchema']
+            const table = data['toTable']
+            let tableData
+            
+            if (table !== null) {
+              if (!schema) {
+                tableData = this.tables.find((t: TableOrView) => escapeHtml(t.name) === table)
+              } else {
```

Comment:
You should just add the async await to the `getColumns` method. Tabulator supports the `valuesLookup` parameter being a promise. See https://github.com/olifolkerd/tabulator/blob/0675efd4b7532d91415b9b5ac3bd11b602ba57cf/src/js/modules/Edit/List.js#L525

it calls `valuesLookup` and if the result is a promise, it awaits.

Reply:
@wmontgomery if you make this change we can merge this.

Reply:
Seeing this now, will be making the change today